tool/grind.dart

```diff
@@ -705,6 +716,83 @@ Future<void> serveFlutterDocs() async {
   ]);
 }
 
+@Task('Serve language test directory docs on port 8004')
+@Depends(buildLanguageTestDocs)
+Future<void> serveLanguageTestDocs() async {
+  log('launching dhttpd on port 8004 for language tests');
+  var launcher = SubprocessLauncher('serve-language-test-docs');
+  await launcher.runStreamed(sdkBin('pub'), ['get']);
+  await launcher.runStreamed(sdkBin('pub'), [
+    'global',
+    'run',
+    'dhttpd',
+    '--port',
+    '8004',
+    '--path',
+    path.join(languageTestPackageDir.path, 'doc', 'api'),
+  ]);
+}
+
+@Task('Build docs for a language test directory in the SDK')
+Future<void> buildLanguageTestDocs() async {
+  var launcher = SubprocessLauncher('build-language-test-docs');
+  if (languageTestPath == null) {
+    fail('LANGUAGE_TESTS must be set to the directory to build language tests');
```

Comment:
This line isn't clear to me. Is it the directory in which language tests are to be built? Or the directory of language tests?

Reply:
fixed